src/rust/storage/seg/tests/integration_basic.rs

```diff
@@ -0,0 +1,283 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use rustcommon_time::*;
+use segcache::*;
```

Comment:
`secache`->`seg`

`cargo test` doesn't build. `SegCache` needs to be updated to `Seg` throughout this file.

Reply:
Fixed. There were quite a few places where the tests failed because of this